### ISSUE TITLE
drivecasa broken with casa 5

### ISSUE THREAD

**Issue by gijzelaerr** [kernsuite/packaging#154]
https://github.com/timstaley/drive-casa/issues/7

**Comment by gijzelaerr** [comment]
a patched casa 5.4.0 has been uploaded to KERN-dev, now only we need to make a new release of drivecasa that is compatible with both versions.

**Comment by gijzelaerr** [comment]
The issue has been reported upstream (if you have access to the tracker):

https://help.nrao.edu/index.php?/Tickets/Ticket/View/13505

Juergen Ott replied:
```
I will have a look, we barely used that option, but indeed it can be useful. 

- juergen 
```

**Comment by gijzelaerr** [comment]
The NRAO issue tracker states this is going to be resolved in CASA 5.8/6.2.

This issue is just too much effort to keep alive, and casa 6 eliminated the need for drive casa. I'm going to close this issue.
